scripts/ci/travis-script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Script that runs in every Travis CI container. The script is responsible for delegating
```

Comment:
I don't see any reason we can't include within it everything we want to re-use between repos.